src/styles/main.scss

```diff
@@ -0,0 +1,194 @@
+@import "~reset-css";
+@import "vars";
+@import "mixins";
+
+body {
+  font-family: "Roboto Condensed", sans-serif;
+  color: $secondry-color;
+}
+
+header {
```

Comment:
Please try to split big components (header, sidebar etc) with small one. And do not forget about BEM. It will help you to split components

Reply:
I will add work with components(component templates) as part of work with the main logic